app/views/comments/_comment.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="well well-sm" style="margin-top: 10px">
+  	<%= comment.user.username %>:
+  	<%= comment.message %><br>
+  	<%= link_to 'Reply', new_comment_path(comment: { meme_id: comment.meme.id, user_id: current_user.id, parent_id: comment.id }) %>
```

Comment:
This causes an error when the user isn't logged in. Two ways to handle this--either don't show the reply button, or rework this to show the 'must be logged in' screen